automation_tools/repository.py

```diff
@@ -145,14 +146,14 @@ def enable_satellite_repos(cdn=False, beta=False, disable_enabled=True,
 
     if disable_enabled is True:
         disable_beaker_repos(silent=True)
-        disable_repos('*', silent=True)
+        disable_repos('*', silent=True, warn_only=True)
 
     repos = [
         'rhel-{0}-server-rpms',
         'rhel-server-rhscl-{0}-rpms',
     ]
     if sat_version == '6.4':
-        repos.append('rhel-{0}-server-ansible-2-rpms')
+        repos.append('rhel-{0}-server-ansible-2.6-rpms')
```

Comment:
@jhutar please add 6.5, we need ansible 2.6 for 6.5 as well.

Reply:
Added if for 6.5. Thanks